requirements.txt

```diff
@@ -0,0 +1,5 @@
+requests
+boto3==1.16.30
+botocore==1.19.30
+dynamodb-json==1.3
+
```

Comment:
we use pipfile to manage all dependency, not requirements.txt. example here: https://github.com/ascending-llc/students_directory